src/collection/style.js

```diff
@@ -71,6 +71,25 @@ var elesfn = ({
     }
   },
 
+  // get the associated values at the specified index when parsing a property with a multiple tag
+  parsedStyleMult: function( property, index ){
+    var ele = this[0];
```

Comment:
If you want to use the default you need to ` || defaultPropertyValue` for each property as it's read (`pfValue`, `value`, etc.).  You can't use the approach you have above, especially because you can't ever modify `style`.  Only the styler does that, and nothing changes the properties once they're created.